src/Aspire.Hosting/DistributedApplicationExecutionContext.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Exposes the global contextual information for this invocation of the AppHost.
+/// </summary>
+public class DistributedApplicationExecutionContext
+{
+    internal DistributedApplicationExecutionContext(DistributedApplicationOperation operation)
+    {
+        Operation = operation;
+    }
```

Comment:
Why is this internal? How are you supposed to test these callbacks?

Reply:
Good point. I didn't notice because we use InternalsVisibleTo for Aspire.Hosting.Tests. Fixed.